x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -237,7 +238,7 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
         deepLinks: [
           {
             id: SecurityPageName.sharedExceptionListDetails,
-            title: 'List Details',
+            title: SHARED_LIST_DETAILS,
             path: '/exceptions/shared/:exceptionListId',
```

Comment:
can you use the `SHARED_EXCEPTION_LIST_DETAIL_PATH` here too?

Reply:
Sure, thanks!